polyfills/document/elementsFromPoint/tests.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-env mocha, browser */
+// eslint-disable-next-line no-unused-vars
+/* globals proclaim */
+
+describe('document.elementsFromPoint', function () {
+    function getElementCenter(el) {
+        var elBounds = el.getBoundingClientRect()
+
+        var x = elBounds.x + elBounds.width / 2;
+        var y = elBounds.x + elBounds.height / 2;
+
+        return [x, y];
+    }
+
+    it('returns all the elements at the specified coordinates', function() {
+        var container = document.body.appendChild(document.createElement('div'));
+        var p = container.appendChild(document.createElement('p'));
+
+        p.innerText = 'Some text';
+
+        var center = getElementCenter(p);
+
+        var elements = document.elementsFromPoint(center[0], center[1]);
+        elements = [].map.call(elements, function(e) {
+            return e.tagName.toLowerCase();
+        });
+        proclaim.deepStrictEqual(elements, ['p', 'div', 'body', 'html']);
```

Comment:
We can't use Array.prototype.map in the tests because IE 8 does not support `Array.prototype.map`.

We can replace it with a for-loop like below:
```suggestion
        var elements = document.elementsFromPoint(center[0], center[1]);
        var elementTagNames = [];
        for (var i = 0; i < elements.length; I++) {
        	elementTagNames.push(element[i].tagName.toLowerCase());
        }
        proclaim.deepStrictEqual(elements, ['p', 'div', 'body', 'html']);
```